spec/models/browse_condition_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe BrowseCondition, type: :model do
+  describe "BrowseCondition mapping" do
+    it "should map BrowseCondition with mesh terms and types" do
```

Comment:
add this:  

it "should map BrowseCondition with mesh terms and types"**, schema: :v2** do